components/AppBar/examples/AppBarManualTabsExample.swift

```diff
@@ -0,0 +1,203 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import CoreGraphics
+
+import MaterialComponents.MaterialAppBar
+import MaterialComponents.MaterialTabs
+
+// An example that demonstrates the behavior of an App Bar with Tabs and manually swapped tab view
+// controllers. This example is distinct from a typical tab bar view controller in that it does not
+// make use of a horizontally-paging scroll view. This example also makes use of the
+// canAlwaysExpandToMaximumHeight API to allow the header to maintain its expanded state when
+// swapping between tabs.
+class AppBarManualTabsExample: UIViewController {
+
+  lazy var appBarViewController: MDCAppBarViewController = self.makeAppBar()
+  var colorScheme = MDCSemanticColorScheme()
+  var typographyScheme = MDCTypographyScheme()
+
+  fileprivate let firstTab = SimpleTableViewController()
+  fileprivate let secondTab = SimpleTableViewController()
+  private var currentTab: SimpleTableViewController? = nil
+
+  lazy var tabBar: MDCTabBar = {
+    let tabBar = MDCTabBar()
+
+    tabBar.items = [
+      UITabBarItem(title: "First", image: nil, tag:0),
+      UITabBarItem(title: "Second", image: nil, tag:1)
+    ]
+
+    tabBar.delegate = self
+    return tabBar
+  }()
+
+  override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
```

Comment:
_Question_: Do we need these initializers or can setting the titles be done in viewDidLoad?

Reply:
As a general practice non-view initialization logic should live in the init method, self.title and navigationItem configuration included.